filebeat/_meta/common.reference.p2.yml

```diff
@@ -258,6 +258,31 @@ filebeat.inputs:
   # The number of seconds of inactivity before a remote connection is closed.
   #timeout: 300s
 
+#------------------------------ Syslog input --------------------------------
+# Experimental: Config options for the Syslog input
+#- type: syslog
+  #enabled: false
+
+  #protocol.tcp:
+    # The host and port to receive the new event
+    #host: "localhost:9000"
+
+    # Character used to split new message
+    #line_delimiter: "\n"
+
+    # Maximum size in bytes of the message received over TCP
+    #max_message_size: 20MiB
+
+    # The number of seconds of inactivity before a remote connection is closed.
+    #timeout: 300s
+
+  #protocol.udp:
```

Comment:
Can one input have tcp and udp at the same time?  If not, we should probably have the syslog input defined twice here.

Reply:
it cannot for complexity reason :) I am OK to have it twice.

Reply:
I'm glad it can't. Perhaps we should have a check to make sure only one is used at the same time.

Reply:
Will take a note to check if we can change that error message.